android/build.gradle

```diff
@@ -21,6 +21,10 @@ allprojects {
         maven {
             url 'https://github.com/uPhyca/stetho-realm/raw/master/maven-repo'
         }
+
```

Comment:
Why this extra space?
Also can you please add some screenshots? :)

Reply:
@the-dagger As the instance of openevent is down for current, I will the run the activity directly and will show how the following is working .